test/python/Utils_t/PortForward_t.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python
+"""
+Unittests for PortForward
+"""
+
+from __future__ import division, print_function
+
+import unittest
+
+from Utils.PortForward import portForward, PortForward
+
+
+class RequestHandler(object):
+    def __init__(self, config=None, logger=None):
+        super(RequestHandler, self).__init__()
+        if not config:
+            config = {}
+
+    @portForward(8443)
+    def request(self, url, params=None, headers=None, verb='GET',
+                verbose=0, ckey=None, cert=None, doseq=True,
+                encode=False, decode=False, cookie=None, uri=None):
+        return url
+
+
+class PortForwardTests(unittest.TestCase):
+    """
+    Unittest for PortForward decorator and class
+    """
+
+    def __init__(self, *args, **kwargs):
+        super(PortForwardTests, self).__init__(*args, **kwargs)
+        self.urlTestList = ['https://cmsweb.cern.ch',
+                            'https://cmsweb.cern.ch/couchdb',
```

Comment:
Todor, a minor request, but something important to always be considered.
To make maintenance of this unit test easier, can you please reduce this list of urls to a short list of representative urls instead? 3 or 6 items should probably be enough.

Reply:
Definitely, consider it done. But I would rather like to have the full set of 3-4 urls in all their three `string`-like types so, U would say the best I can reduce this list is to a dozen, but still much better than 80.

Reply:
Done